### PR TITLE
Add error logs

### PR DESCRIPTION
**Pull request by DLarson-Oddball**
## Summary

- Adding a few logs to aid in better understanding why some VR&E claims are not being processed.

## Related issue(s)
- [department-of-veterans-affairs/va.gov-team#0000](https://github.com/department-of-veterans-affairs/va.gov-team/issues/65513)

## Testing done

- Submit form 28-1900 and assert successful submissionl

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by tblackwe — APPROVED** [review]
Can you update CODEOWNERS to include @department-of-veterans-affairs/benefits-non-disability for `app/models/saved_claim/veteran_readiness_employment_claim.rb` ?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -312,6 +312,7 @@ app/models/prescription.rb @department-of-veterans-affairs/vfs-mhv-medications @
 app/models/rate_limited_search.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/saml_request_tracker.rb @department-of-veterans-affairs/vsp-identity
 app/models/saved_claim @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+app/models/saved_claim/veteran_readiness_employment_claim.rb @department-of-veterans-affairs/benefits-non-disability
```
Comment (tblackwe) [inline .github/CODEOWNERS:315]: Github doesnt honor file heirarchy, so needs to be 
`app/models/saved_claim/veteran_readiness_employment_claim.rb @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group`

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -312,7 +312,7 @@ app/models/prescription.rb @department-of-veterans-affairs/vfs-mhv-medications @
 app/models/rate_limited_search.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/models/saml_request_tracker.rb @department-of-veterans-affairs/vsp-identity
 app/models/saved_claim @department-of-veterans-affairs/Benefits-Team-1 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/models/session.rb @department-of-veterans-affairs/vsp-identity
+app/models/saved_claim/veteran_readiness_employment_claim.rb @department-of-veterans-affairs/benefits-non-disability @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
```
Comment (tblackwe) [inline .github/CODEOWNERS:315]: 😭 Sorry 1 more comment.
`app/models/session.rb @department-of-veterans-affairs/vsp-identity` got deleted

**Review by tblackwe — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]
The codeowners file was complaining about the team @department-of-veterans-affairs/benefits-non-disability. I added the team to vets-api as collaborators and gave yall write permissions, so now the file is valid 😄 

LGTM.

**Comment by rmtolmach** [comment]
btw, I merged some rspec changes on Friday. I wanted to be sure your branch was up-to-date so it would catch any errors pre-merge, so that's why I merged master into your branch.

**Pull request merged**